src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -27,7 +27,7 @@ internal abstract class SourceDelegateMethodSymbol : SourceMemberMethodSymbol
             : base(delegateType, syntax.GetReference(), location: syntax.Identifier.GetLocation(), isIterator: false)
         {
             _returnType = returnType;
-            this.MakeFlags(methodKind, declarationModifiers, _returnType.IsVoidType(), isExtensionMethod: false);
+            this.MakeFlags(methodKind, declarationModifiers, _returnType.IsVoidType(), isExtensionMethod: false, isNullableEnabled: false);
```

Comment:
>isNullableEnabled: false [](start = 113, length = 24)

Similarly, any way to assert this is never depended on?

Reply:
The value can be depended on since nullable analysis should be disabled for delegate methods.

---
In reply to: [546866397](https://github.com/dotnet/roslyn/pull/49953#discussion_r546866397) [](ancestors = 546866397)

Reply:
Consider overriding the method in this type and throwing unreachable.

---
In reply to: [547746250](https://github.com/dotnet/roslyn/pull/49953#discussion_r547746250) [](ancestors = 547746250,546866397)

Reply:
We shouldn't crash the process if there is a caller to `IsNullableAnalysisEnabled()`.

---
In reply to: [551487187](https://github.com/dotnet/roslyn/pull/49953#discussion_r551487187) [](ancestors = 551487187,547746250,546866397)